backup_test.go

```diff
@@ -310,14 +310,14 @@ func TestBackup(t *testing.T) {
 
 		defer removeDir(tmpdir)
 		opt := DefaultOptions(filepath.Join(tmpdir, "backup0"))
-		runBadgerTest(t, &opt, func(t *testing.T, db *DB) {
+		runBadgerTestParallel(t, &opt, func(t *testing.T, db *DB) {
```

Comment:
Why change `runBadgerTest` to `runBadgerTestParallel`? We could run all tests in parallel, always.

Reply:
Some tests need to be run sequentially TestConflict, TestGoroutineLeak